riff-raff.yaml

```diff
@@ -28,7 +28,7 @@ deployments:
         CODE: PinBoardStack-CODE.template.json
         PROD: PinBoardStack-PROD.template.json
       amiTags:
-        Recipe: editorial-tools-focal-java8-ARM-WITH-cdk-base
+        Recipe: editorial-tools-jammy-java11-ARM-WITH-cdk-base
```

Comment:
suggest using `editorial-tools-jammy-java11` instead - from a look at the recipe they should be essentially equivalent, but with a more rationalised name now that ARM and cdk-base are standard in our recipes